lib/velum/kubernetes.rb

```diff
@@ -22,7 +22,7 @@ def self.kubeconfig
                                           targets:     "roles:kube-master",
                                           target_type: "grain",
                                           arg:         "cat /etc/pki/minion.key"
-      host = Minion.master.applied.pluck(:fqdn).first
+      host = Pillar.value(pillar: :apiserver) || Minion.master.applied.pluck(:fqdn).first
```

Comment:
It looks like we've added the new field as required, so the pillar value should always exist. Should we remove the `|| Minion.master.applied.pluck(:fqdn).first`?